### PR TITLE
refactor(esl-media): event resubscribtion and resize target

### PR DESCRIPTION
**Pull request by fshovchko**

### CODE REVIEW

**Review by ala-n — APPROVED** [review]

File: src/modules/esl-media/core/esl-media.ts [hunk 1]
```diff
@@ -327,7 +326,7 @@ export class ESLMedia extends ESLBaseElement {
 
   @listen({
     event: 'resize',
-    target: window,
+    target: ESLResizeObserverTarget.for,
     condition: ($this: ESLMedia) => $this.fillModeEnabled
   })
   @decorate(rafDecorator)
```
Comment (ala-n) [inline src/modules/esl-media/core/esl-media.ts:332]:
```suggestion
```

**Review by yadamskaya — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Review by ala-n — COMMENTED** [review]
There are extra imports reported by eslint

**Review by ala-n — APPROVED** [review]

**Review by Natalie-Smirnova — APPROVED** [review]

**Pull request merged**